solutions/swb-reference/jest.config.js

```diff
@@ -24,8 +24,18 @@ async function init() {
     fs.readFileSync(join(__dirname, `integration-tests/config/${stage}.yaml`), 'utf8')
   );
 
+  const apiStackOutputs = JSON.parse(
```

Comment:
could we try catch and show a friendly error in case they don't have the json file